src/concrete_convex_hull.jl

```diff
@@ -304,13 +309,15 @@ function _convex_hull_1d!(points::Vector{VN})::Vector{VN} where {N<:Real, VN<:Ab
 end
 
 function _convex_hull_nd!(points::Vector{VN};
-                          backend=nothing
+                          backend=nothing,
+                          solver=nothing
                           )::Vector{VN} where {N<:Real, VN<:AbstractVector{N}}
     V = VPolytope(points)
     if backend == nothing
         backend = default_polyhedra_backend(V, N)
+        solver = default_lp_solver(N)
```

Comment:
`backend == nothing` does not imply `solver == nothing`. Maybe just keep it as it is and resolve the case `solver == nothing` in the other place (see my previous comment).

Reply:
That's correct, the user could have passed a solver but not a backend. Defaults for both options (`backend` and `solver`) are handled by `remove_redundant_vertices` so i opt for that case and remove the check in `_convex_hull_nd!` altogether.

Reply:
> that is not possible.

Why not? And why set the `solver` only if `backend == nothing`?

Reply:
> Why not?

It is not possible without changing `remove_redundant_vertices`, sure. 

> And why set the solver only if backend == nothing?

It is a choice if the `backend` is not set then one needs a solver because we are using the default backend, but the converse is not always true (for example if the backend was chosen to be `CDDLib`).

Reply:
Sounds like we should pass `backend` to `default_lp_solver` then.

Reply:
Generally we should not set the default backend/solver in different functions (unless there is a good reason why some functions use a different default than others). There should be a central place where we do that, and only immediately before we actually use it. This makes it easier to avoid inconsistencies and make changes in the future.